commands/station.js

```diff
@@ -44,6 +44,9 @@ export const station = async ({ json, experimental }) => {
   ) {
     panic('FIL_WALLET_ADDRESS must start with f410 or 0x')
   }
+  if (FIL_WALLET_ADDRESS.startsWith('0x') && !isEthAddress(FIL_WALLET_ADDRESS)) {
```

Comment:
We are converting f4 addresses to 0x format few lines below.

```js
  const ethAddress = FIL_WALLET_ADDRESS.startsWith('0x')
    ? FIL_WALLET_ADDRESS
    : ethAddressFromDelegated(FIL_WALLET_ADDRESS)
```

I propose to move the validation down there and validate the final `ethAddress` value that we pass to other components.

Reply:
I thought about this too, but it's testing something else: It's testing whether the passed or the converted address is valid. As it is now, it's only testing whether the passed address is valid. I have a concern that the proposed method could shadow some tricky to find bugs.